ingester/ingester.go

```diff
@@ -0,0 +1,24 @@
+package ingester
+
+import (
+	"fmt"
+	"github.com/log-yarder/yarder/discovery"
+)
+
+// Ingester is the entry point for log entry ingestion. It fans the entries out
+// to an appropriate number of appenders.
+type Ingester struct {
+	Discovery *discovery.Discovery
+}
+
+// HandleIngest ingests a single log entry.
+func (i *Ingester) HandleIngest(entry string) error {
+	// For now, just append to all appenders.
+	for _, appender := range i.Discovery.Appenders {
+		err := appender.HandleAppend(entry)
+		if err != nil {
+			return fmt.Errorf("Unable to handle append: %v", err)
```

Comment:
The error wrapping would be more useful if it gave info about the loop state, i.e. the index for now (some name for the appender in the future would be nice). Also I just remembered that https://github.com/golang/go/wiki/CodeReviewComments#error-strings suggests lowercase error strings for cleaner concatenation.

Reply:
Good points. Lower cased and added loop index.